### PR TITLE
chore: add decision about market-depth with regard to iceberg orders

### PR DESCRIPTION
**Pull request by wwestgarth**
In response to:; https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1686217649242679

### CODE REVIEW

File: protocol/0014-ORDT-order_types.md [hunk 1]
```diff
@@ -272,6 +274,7 @@ Network orders are used during [position resolution](./0012-POSR-position_resolu
 
 1. API end points should be available to query initial peak size, minimum peak size, quantity, displayed quantity and remaining (<a name="0014-ORDT-036" href="#0014-ORDT-036">0014-ORDT-036</a>)
 2. The additional fields relating to iceberg orders should be available in the streaming api end points
```
Comment (gordsport) [inline protocol/0014-ORDT-order_types.md]:
```suggestion
2. The additional fields relating to iceberg orders should be available in the streaming api end points (<a name="0014-ORDT-040" href="#0014-ORDT-040">0014-ORDT-040</a>)
```

File: protocol/0014-ORDT-order_types.md [hunk 2]
```diff
@@ -272,6 +274,7 @@ Network orders are used during [position resolution](./0012-POSR-position_resolu
...
+3. API end points showing market-depth or price-level volume should include the full volume of iceberg orders (<a name="0014-ORDT-039" href="#0014-ORDT-039">0014-ORDT-039</a>)
```
Comment (gordsport) [inline protocol/0014-ORDT-order_types.md]:
```suggestion
3. API end points showing market-depth or price-level volume should include the full volume of iceberg orders (<a name="0014-ORDT-041" href="#0014-ORDT-041">0014-ORDT-041</a>)
```

**Review by gordsport — CHANGES_REQUESTED** [review]
@wwestgarth - I have made a few suggestions to fix the approbation. If the first change is accepted I will close this open PR: https://github.com/vegaprotocol/specs/pull/1742

File: protocol/0014-ORDT-order_types.md [hunk 3]
```diff
@@ -169,6 +169,8 @@ If the remainders of multiple icebergs sit at the same price and are not fully u
 
 * An iceberg order refresh must generate an event of the event bus.
 
+* Any API that returns information about market-depth or the orderbook volume will include an iceberg orders full volume and not just its `display quantity`.
```
Comment (witgaw) [inline protocol/0014-ORDT-order_types.md]:
```suggestion
* Any API that returns information about market-depth or the orderbook volume will include an iceberg order's full volume and not just its `display quantity`.
```

**Review by witgaw — DISMISSED** [review]

**Review by witgaw — APPROVED** [review]

**Review by gordsport — APPROVED** [review]

**Pull request merged**